include/openmapper/input_source.h

```diff
@@ -28,7 +28,7 @@ class InputSource {
                   // ROS, native mobile framework...
   };
   InputSource();
-  virtual ~InputSource();
+  // virtual ~InputSource();
```

Comment:
I had to comment out those virtual destructors (see below too) in order to build it for iOS. do those currently have any crucial function?